Logger/Book.cs

```diff
@@ -0,0 +1,11 @@
+namespace Logger;
+
+public record class Book(string Title) : BaseEntity
+{
+    //Implimented implicitly since Name isn't causing any collisions, and if we ever need an Author name we would
+    // probably add an Author property of Entity type person which would get the Author name through that.
+    public override string Name { 
+    get=>$"{Title}";}
```

Comment:
```suggestion
    public override string Name { get=> Title; }
```
this should simplify this to not need an interpolated string